include/zephyr/net/wifi_mgmt.h

```diff
@@ -1303,6 +1310,14 @@ struct wifi_mgmt_ops {
 	 * @return 0 if ok, < 0 if error
 	 */
 	int (*wps_config)(const struct device *dev, struct wifi_wps_config_params *params);
+	/** Start AP WPS PBC/PIN
+	 *
+	 * @param dev Pointer to the device structure for the driver instance
+	 * @param params wps operarion parameters
+	 *
+	 * @return 0 if ok, < 0 if error
+	 */
+	int (*ap_wps_config)(const struct device *dev, struct wifi_wps_config_params *params);
```

Comment:
the structure `wifi_wps_config_params` is same, so, why not use existing `wps_config` for AP too?

Reply:
yes, can use the same